containers/clojure/README.md

```diff
@@ -0,0 +1,134 @@
+# Clojure
```

Comment:
This would be a community definition, so we'd want (Community) next to the name here.